Dockerfile.slim

```diff
@@ -0,0 +1,49 @@
+#---------------------------------------------------------------------------------------------
+# See LICENSE in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+ARG PYTHON_VERSION="3.11"
+
+FROM python:${PYTHON_VERSION} as builder
+
+WORKDIR /wheels
+
+RUN pip install --upgrade pip \
+    && pip install wheel
+
+COPY . .
+
+RUN pip wheel .
+
+RUN mkdir /wheels/recommended_plugins
+
+COPY ./recommended-plugins.txt ./recommended-plugins.txt
+
+RUN pip wheel .[recommended-plugins] --wheel-dir=/wheels/recommended_plugins
+
+FROM python:${PYTHON_VERSION}-slim
+
+# See http://label-schema.org for metadata schema
+# TODO: Add `build-date` and `version`
+LABEL maintainer="ApeWorX" \
+      org.label-schema.schema-version="2.0" \
+      org.label-schema.name="ape-slim" \
```

Comment:
this may need to be a parameter, as it will be the same otherwise for both slim and fat

Reply:
I think it should be the same for both

Reply:
Ya that makes sense. Alternatively, you can exclude from the slim